### PR TITLE
Add property to sac to amend the timeout for generating each individual secret.

### PR DESCRIPTION
**Pull request by lee-baines**
Add property to sac to amend the timeout for generating each individual secret.

Closes: #169

### CODE REVIEW

**Review by jrcast — APPROVED** [review]
LGTM

File: api/v1alpha1/secretagentconfiguration_types.go [hunk 1]
```diff
@@ -191,6 +191,9 @@ type AppConfig struct {
 	AWSRegion             string         `json:"awsRegion,omitempty"`
 	AzureVaultName        string         `json:"azureVaultName,omitempty"`
 
+	// Optional timeout value to generate a individual secret. Defaults to 40
+	SecretTimeout *int `json:"secretTimeout,omitempty"`
```
Comment (maxres-fr) [inline api/v1alpha1/secretagentconfiguration_types.go:195]: Can you also add this to:
* sample config in _this_ repo
* secret_agent_config.yaml - in forgeops

Comment (lee-baines) [reply api/v1alpha1/secretagentconfiguration_types.go:195]: yes, will do

Comment (lee-baines) [reply api/v1alpha1/secretagentconfiguration_types.go:195]: Its in the sample config but commented out
`    ## Timeout in seconds for generating each individual secret. Default 40
    # secretTimeout: 40 `

Comment (maxres-fr) [reply api/v1alpha1/secretagentconfiguration_types.go:195]: sorry about that, is it in forgeops too?

Comment (lee-baines) [reply api/v1alpha1/secretagentconfiguration_types.go:195]: I've created a separate ticket for that, I'll do that today

**Review by maxres-fr — APPROVED** [review]

**Pull request merged**